app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1>This is an individual user</h1>
+<section class="container">
+
+<article class="user-card">
+<img src="<%= @user.photo %>" class="user-photo" alt="user photo">
+<div class="user-card-info">
+<h2><%= @user.name %></h2>
+<p><%= link_to "Number of posts:", user_posts_url(@user.id) %> <%= @user.posts_count %></p>
+</div>
+</article>
+<div class="user-bio">
+  <strong>Bio</strong>
+  <p><%= @user.bio %></p>
+</div>
+<% if @user.recent_posts.any? %>
+<% @user.recent_posts.each do |post| %>
+<div class="post-details">
+  <h2><%= link_to "Post #", user_post_url(@user.id, post.id) %><%= post.id %></h2>
+  <h3><%= post.content %></h3>
+  <p>Comments:<%= post.commentsCounter || 0 %>, Likes:<%= post.likes_count || 0 %></p>
+</div>
+<% end %>
+<%= link_to user_posts_path(@user.id) do %>
+<% end %>
+<button type="button" class="see-posts">See all posts</button>
+<% else %>
+<p>No posts found.</p>
```

Comment:
- [ ] The see-all post button should take the user to the `user's post index` page. Kindly do so.